addons/web_editor/static/src/xml/snippets.xml

```diff
@@ -15,7 +15,10 @@
                         <!-- Button-like handle to drag and drop -->
                         <div class="o_move_handle"/>
                     </div>
-                    <button type="button" class="oe_snippet_remove fa fa-trash"/>
+                    <div class="o_overlay_edit_options">
```

Comment:
Not sure we'll need another section, the option can put his button in `o_overlay_move_options` just like the `SnippetMove` option :+1:

Reply:
A container `o_overlay_edit_options` allows us to display the options replace and delete options the same way as `o_overlay_move_options` even if the move options are not displayed, but maybe there is a cleaner implementation

Reply:
As I said, I was not sure, you're right of course :+1: